src/Components/PendingRequests/PendingRequests.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import { gql, useQuery } from '@apollo/client'
+import Request from "../Request/Request";
+
+const REQUESTS = gql `
+  query user {
+    user(id:1) {
```

Comment:
I believe we will need to make this id dynamic. I can show you what we did for book details

Reply:
We talked about this in slack thread -- can't make this dynamic until we have Josh's contribution of saving the user in app